app/index.ejs

```diff
@@ -211,11 +211,10 @@
             }
         });
         if(domain == 'cbd.int'){
-            window.ga=window.ga||function(){(ga.q=ga.q||[]).push(arguments)};ga.l=+new Date;
-            
-            ga('create', 'UA-1996573-14', 'auto');
-            ga('set',    'hostname', window.location.hostname);
-            ga('set',    'location', undefined);
+            window.dataLayer = window.dataLayer || [];
+            function gtag(){dataLayer.push(arguments);}
+            gtag('js', new Date());
+            gtag('config', 'G-5WHL82Y4KD');
```

Comment:
can use env variable here too?

Reply:
It was included on `process.env.GOOGLE_ANALYTICS_CODE`.